### DISCUSSION TITLE
Global feed filters

### DISCUSSION THREAD

**Discussion by Jamiras** [RetroAchievements/RAWeb#699] [Ideas]
There are enough players now that the global feed is very noisy with people logging in and starting to play games. These notifications should be removed from the global feed.

**Comment by kdecks** [comment]
I wouldn't mind seeing a reimagining of the global feed. Is it previously worked with live updates it had a much nicer appeal but as auto updating is disabled this doesn't have the same effect as it used to

**Comment by meleu** [comment]
I don't agree on removing "started to play". It's useful when investigating cheaters.

**Comment by meleu** [comment]
It would be cool if we could customize the filters.
